benchutils/tests/test_metrics.py

```diff
@@ -92,31 +93,80 @@ def test_rmse_basic(self):
 
 
 # TODO test these (blocked by unittest filesystem)
-class TestProfileError(unittest.TestCase):
+class TestProfileError(testbase.BaseTestCase):
+
+    def setUp(self):
+        super(TestProfileError, self).setUp()
+        self.exp = self.get_data_path('expected_genus_profile.txt')
+        self.obs1 = self.get_data_path('sample_genus_profile1.txt')
+        self.obs2 = self.get_data_path('sample_genus_profile2.txt')
 
     def test_errors_metric_not_in_available_metrics(self):
-        pass
+        with self.assertRaisesRegex(ValueError, 'not in available metrics'):
+            profile_error('foo', 'bar', 'baz', rank='genus',
```

Comment:
Looks like this `profile_error` generates a report file, but these test functions didn't test the content of that file. Is this okay?

Reply:
It would be _ideal_ if I check the contents. For now, the values are checked on the internal functions and these tests more show that no changes are causing errors in the exceptions... I'm going to kick this one down the road to a separate issue.